guide/variables.md

```diff
@@ -43,14 +43,14 @@ time-elapsed = Time elapsed: { $duration }s.
 ```
 
 For the `$duration` variable value of `12345.678`, the following message will be
-displayed in the product, assuming the language is set to British or American
+displayed in the product, assuming the language is set to British, Canadian, or American
```

Comment:
```suggestion
displayed in the product, assuming the language is set to British or American
```

Given the discussion in #234, let's avoid mentioning Canada here. The specific cases we're comparing don't really matter, as the overall point is that It's Complicated, and that's actually easier to make by leaving out Canada, which itself Is Complicated.